app/assets/stylesheets/govuk_publishing_components/components/_add-another.scss

```diff
@@ -0,0 +1,6 @@
+@import "govuk_publishing_components/individual_component_support";
+@import "govuk/components/button/button";
```

Comment:
I think some additional styles need to be imported for:
```
@import "govuk/components/checkboxes/checkboxes";
@import "govuk/components/input/input";
@import "govuk/components/label/label";
```

Reply:
The checkboxes and inputs are provided by the caller of the component, so it's up to whoever calls the component to ensure that they are covered (the component could also contain other input types such as radio buttons or something entirely custom)

Reply:
In this case it's similar to the the other inputs.

Whilst we expect the user to pass in a checkbox - it doesn't necessarily mean that they're using the govuk_publishing_components checkbox, they may want to style checkboxes in a particular way.

In the case that they are using the govuk_publising_components checkbox they'll be calling the render so should also be responsible for ensuring that the checkbox style is included - rather than am implicit dependancy through this component.

Happy to chat about it more if you want or if you still think we need the dependancy here, I can add it, just wanted to explain why I think it's better not to have it.